src/FileIndexer.ts

```diff
@@ -14,27 +14,23 @@ import {
   typeParameterDescriptor,
 } from './Descriptor'
 import { Input } from './Input'
-import * as lsif from './lsif'
-import { LsifSymbol } from './LsifSymbol'
-import { lsiftyped } from './main'
 import { Packages } from './Packages'
 import { Range } from './Range'
+import * as scip from './scip'
+import { ScipSymbol } from './ScipSymbol'
```

Comment:
nit: Following JS/TS conventions, I think this should be called SCIPSymbol, similar to HTML and URL in browser APIs.

Reply:
I recall seeing in relation to `XMLHttpRequest` that the convention is that only three-letter acronym should remain uppercase. I really dislike SCIPSymbol, it's easier IMO to read ScipSymbol so I prefer to keep it